gcp/table_gcp_storage_object.go

```diff
@@ -255,7 +265,7 @@ func listStorageObjects(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
 		}
 	}
 
-	resp := service.Objects.List(bucket).Projection("full").MaxResults(*maxResults)
+	resp := service.Objects.List(bucket).Prefix(prefix).Projection("full").MaxResults(*maxResults)
```

Comment:
@villers If no prefix is passed in, e.g., `select * from gcp_storage_object`, does this API call still work as it did before the change?

Reply:
Yes, it will list the objects without considering the prefix if it's empty.